webapp/channels/src/components/size_aware_image.tsx

```diff
@@ -214,6 +216,18 @@ export default class SizeAwareImage extends React.PureComponent<Props, State> {
             ariaLabelImage += ` ${fileInfo.name}`.toLowerCase();
         }
 
+        const fileType = getFileType(fileInfo?.extension ?? '');
+
+        let conditionalSVGStyleAttribute = {};
+        if (fileType === FileTypes.SVG) {
+            conditionalSVGStyleAttribute = {
+                style: {
+                    width: dimensions?.width || MIN_IMAGE_SIZE,
+                    height: 'auto',
+                },
+            };
+        }
```

Comment:
Why creating an object to then deconstructing, and not creating the style object itself? Something like this:
```suggestion
        let conditionalSVGStyleAttribute;
        if (fileType === FileTypes.SVG) {
            conditionalSVGStyleAttribute = {
                    width: dimensions?.width || MIN_IMAGE_SIZE,
                    height: 'auto',
            };
        }
```

Reply:
I did the destructuring because otherwise the style object could be an empty object if the file type was not SVG. But thank you for the suggestion, I didn't think of not assigning something to `conditionalSVGStyleAttribute` originally.

Updated it!